pyonfx/shape.py

```diff
@@ -296,7 +297,7 @@ def move(self, x: float = None, y: float = None) -> Shape:
             y = 0
 
         # Update shape
-        self.map(lambda cx, cy: (cx + x, cy + y))
+        self.map(lambda cx, cy, typ: (cx + x, cy + y))
```

Comment:
There is no need to construct a lambda with 3 params here, as you don't use the `typ`.

Reply:
I added it because mypy and Pylance complained about it. Maybe this `self.map(lambda cx, cy, _: (cx + x, cy + y))` could be better?

Reply:
It should be solved by specifying the correct typing as in https://github.com/CoffeeStraw/PyonFX/pull/33#discussion_r682208970